devel/to-f90/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem		1.0
+PortGroup		compilers 1.0
+
+name			to-f90
+version			2000-03-21
+revision		1
+categories		devel
+platforms		darwin
+license			Noncommercial
+maintainers		{@kamischi web.de:karl-michael.schindler} openmaintainer
+homepage		http://cd.textfiles.com/simtel/simtel0101/simtel/fortran/
+master_sites	${homepage}
+distname		to_f90
```

Comment:
Since this filename does not contain the version, please use the unversioned distfile recipe: https://trac.macports.org/wiki/PortfileRecipes#unversioned-distfiles

Reply:
OK